doc/model_persistence.rst

```diff
@@ -4,6 +4,47 @@
 Model persistence
 =================
 
+.. list-table:: Summary of model persistence methods
+   :widths: 25 50 50
+   :header-rows: 1
+
+   * - Persistence method
+     - Pros
+     - Risks / Cons
+   * - :ref:`ONNX <onnx_persistence>`
+     - * Serve models without a Python environment
+       * Serving and training environments independent of one another
+       * Most secure option
+     - * Not all scikit-learn models are supported
+       * Custom estimators are hard to add
+       * Original Python object is lost and cannot be reconstructed
+   * - :ref:`skops_persistence`
+     - * Pickle free and more secure
+       * Contents can be partly validated without loading
+     - * Not as fast as `pickle` based formats
+       * Supports less types than `pickle` based formats
+       * Requires the same environment as the training environment
+   * - :mod:`pickle`
+     - * Native to Python
+       * Can serialize most Python objects
+       * Efficient memory usage with `protocol=5`
+     - * Loading can execute arbitrary code
+       * No memory mapping
```

Comment:
I wonder if we should put this here. Since joblib is the only tool that supports memory mapping (as far as I know), we would have to put "no memory mapping" as a con to all alternatives instead of singling on out.

Reply:
Fair, removed.